doc/news/changes/incompatibilities/20201130LucaHeltai

```diff
@@ -0,0 +1,4 @@
+Changed: GridTools::find_active_cell_around_point() no longer throws an exception, but returns an invalid iterator.
+User codes should check that the returned cell is valid instead of relying on exceptions. This solves issues with MPI, and HDF5.
```

Comment:
We decided that the problem isn't actually with MPI or HDF5 but with not checking exceptions. Let's just omit this part.
```suggestion
User codes should now check that the returned cell is valid instead of relying on exceptions, instead of trying
to catch exceptions that this function may have thrown.
```